source/docs/being-a-team/how-to-ask-effective-questions.rst

```diff
@@ -0,0 +1,69 @@
+.. include:: <isonum.txt>
+
+How to Ask Effective Questions
+==============================
+
+Asking questions effectively is one of the most important skills to develop, especially for your life outside of robotics.
+Being able to learn from your questions properly is crucial and as such, you should be able to ask good questions, help those who want to help you, and make the most of what you are taught.
+
+Research First
+--------------
+
+Before you ask a question, you should be doing your own research to find the answer.
+This will take more time, it will take more effort, but it is a crucial skill to learn to find solutions on your own, and learn how to research effectively to find solutions to your problems.
+In the end, this research will pay off in the long run, helping you to learn about topics relevant to your question, and often helping you to understand the roots of the problems you encounter.
+
+Here are some steps that you could take to research your problem before asking someone a question about your current problem.
+
+- **Google** - Simply search your problem.
+  You will be surprised by how often this can quickly solve your problem although you may not have expected it to.
+  Try word the question in many different ways to find similar questions that may have already been asked.
```

Comment:
Maybe mention stuff like searching using keywords and specific error messages here?